### PR TITLE
fix(loader): fix bug in configmap

### PR DESCRIPTION
**Pull request by guangrui-cloudnatix**

### CODE REVIEW

File: deployments/loader/templates/configmap.yaml [hunk 1]
```diff
@@ -33,9 +33,8 @@ data:
     {{- range .Values.baseModels }}
     - {{ . }}
     {{- end }}
-    models:
-    {{- range .Values.models }}
-    - {{ . }}
+    {{- if .Values.models}}
```
Comment (kkaneda) [inline deployments/loader/templates/configmap.yaml:36]: should we use `with`?

**Review by kkaneda — APPROVED** [review]

File: deployments/loader/templates/configmap.yaml (hunk 1 above)
Comment (kkaneda) [reply deployments/loader/templates/configmap.yaml:36]: sorry for my bug!

probably better to have:

```
    {{- with .Values.models }}
    models:
    {{- toYaml . | nindent 4 }}
    {{- end }}
```

Comment (guangrui-cloudnatix) [reply deployments/loader/templates/configmap.yaml:36]: `if` works too for this case?

Comment (kkaneda) [reply deployments/loader/templates/configmap.yaml:36]: no strong opinion!

**Pull request merged**